### ISSUE TITLE
Sentry.io integration

### ISSUE THREAD

**Issue by rentalhost** [HeidiSQL/HeidiSQL#626]
I have the slightest sense of having already spoken of this subject with you, but since I could not find anything about it, let's try again.

There is a tool called [sentry.io](https://sentry.io) that makes easier to get and handle exceptions that occur in applications. The easiness is: 1. it is free up to a certain level (but it will hardly be reached); 2. facilitates the identification of exceptions without the user reporting; 3. you can identify which exceptions are more serious and more constant than others, and fix them first.

The problem is that it is not implemented for Delphi, but seems easy to create an port via Restful. There are an article [about it](https://medium.com/@Zawuza/collect-exceptions-from-delphi-applications-using-sentry-eb90e689f76b).

Basically it will works together with the madExcept (that seems to have a 5.0 new version, but it should be another topic hehe).

**Comment by ansgarbecker** [comment]
Will keep the current madExcept integration, which also allows uploading crash reports and even an optional screenshot. Exceptions are then stored in a MariaDB table on heidisql.com. A cronjob aggregates these reports so I can see the same exceptions with their number of occurrences, which expresses the importance of them. Once in a while I am checking the top ten of these.
